make.ps1

```diff
@@ -293,7 +293,7 @@ switch ($Command.ToLower())
             $numTestSuitesRun += 1;
 
             $runOutDir = "$outDir\runner-tests\$runConfig"
-            $debugFlag = if ($runConfig -eq 'debug') { '--debug' } else { '' }
```

Comment:
This is a fix for PowerShell 7.3.0

Reply:
seems like this should be its own PR. ill happily merge such a PR quickly so that you could rebase this on it.